### PR TITLE
feat: global time is updated

### PR DESCRIPTION
**Pull request by palashgdev**
Close https://github.com/SigNoz/signoz/issues/1915

### CODE REVIEW

**Review by nityanandagohain — COMMENTED** [review]
- [x] Go to latest is working fine

File: frontend/src/container/LogControls/index.tsx [hunk 1]
```diff
@@ -28,29 +30,72 @@ function LogControls(): JSX.Element | null {
...
+			payload: {
+				maxTime,
+				minTime,
+			},
 		});
 	};
 
 	const handleNavigatePrevious = (): void => {
+		const { maxTime, minTime } = getMinMax(
+			globalTime.selectedTime,
```
Comment (nityanandagohain) [inline frontend/src/container/LogControls/index.tsx:73]: Prev/Next Should not update time range

Comment (nityanandagohain) [reply frontend/src/container/LogControls/index.tsx:73]: Also `X/page`  as Ankit Mentioned

Comment (palashgdev) [reply frontend/src/container/LogControls/index.tsx]: done

**Comment by ankitnayan** [comment]
@palashgdev do the start and end refresh on any action to the searchbox? Either from `close` or `search` button. Or searchbox dropdown builder or any filtering done by selecting a selected field. If not, we should refresh on such actions

**Comment by srikanthccv** [comment]
@palashgdev couple of things while checking with the latest dev

- https://www.loom.com/share/5691f39c77f74ed3b7a8658f1ba2bcc9
- https://www.loom.com/share/06de01836cda4a46bc9c1a5bfa3825f2
- https://www.loom.com/share/a6fc721cc05b44e39572841fd3f69755

**Review by nityanandagohain — COMMENTED** [review]
Functionality Test

- [x] Prev/Next should not update time
- [x] Filtering by clicking on the UI updates time range
- [x]  Search updates the time range
- [x] Interesting/selected field doesn't update time range.
- [x] Go to latest updates the time range

But if I am on nth page, by clicking prev and next I think we should not update timerange if users filter by clicking. But it might be hard to implement.
I think with this release we can go ahead with this changes and see how our users react. cc @ankitnayan 

@palashgdev let me know if there is anything else to test

**Review by nityanandagohain — APPROVED** [review]

**Pull request merged**